manifests/g/Google/GCPW/70.103.49152/Google.GCPW.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Google.GCPW

```

```suggestion
PackageIdentifier: Google.CredentialProviderForWindows
```

It is usually best to not use abbreviations in PackageIdentifiers to avoid confusion. After changing this, the folder and files will need to be renamed